spec/forms/hub/update13614c_form_page2_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe Hub::Update13614cFormPage2 do
+  let(:intake) {
+    build :intake,
+          :with_contact_info,
+          email_notification_opt_in: "yes",
+          state_of_residence: "CA",
+          preferred_interview_language: "en",
+          primary_ssn: "123456789",
+          primary_tin_type: "ssn",
+          signature_method: "online",
+          cv_p2_notes_comments: nil
+  }
+  let!(:client) { Hub::ClientsController::HubClientPresenter.new(create :client, intake: intake) }
+  let(:form_attributes) do
+    { had_wages: "yes",
+      job_count: 1,
+      had_tips: "yes",
+      had_retirement_income: "no",
+      had_disability_income: "yes",
+      had_social_security_income: "no",
+      had_unemployment_income: "yes",
+      had_local_tax_refund: "no",
+      had_interest_income: "no",
+      cv_p2_notes_comments: "money in the banana stand"
+    }
+  end
+
+  describe "#save" do
+    it "persists valid changes" do
+      expect do
+        form = described_class.new(client, form_attributes)
+        form.save
+        intake.reload
+      end.to change(intake, :had_wages).to "yes"
```

Comment:
how come `had_wages` is treated differently from all the other attributes?

Reply:
I could do a long chain of `.and change(`  for the rest of the changes but I decided to use the other pattern and just establish that at least one had changed on the form save; I could just change to use one or the other if its confusing though